modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/fragments/group/commerce/fragments/order-tabs/index.js

```diff
@@ -0,0 +1,169 @@
+	)
+);
+
+const tabPanelItems = Array.from(
+	fragmentElement.querySelectorAll(
+		'[data-fragment-namespace="' + fragmentNamespace + '"].tab-panel-item'
+	)
+);
+
+const persistedTab = (function () {
+	if (!configuration.persistSelectedTab) {
+		let persistedId;
+
+		return {
+			getId() {
+				return persistedId;
+			},
+
+			setId(nextId) {
+				persistedId = nextId;
+			},
+		};
+	}
+
+	return {
+		getId() {
+			return Number(
+				Liferay.Util.SessionStorage.getItem(
+					persistedTabKey,
+					Liferay.Util.SessionStorage.TYPES.PERSONALIZATION
+				)
+			);
+		},
+
+		setId(id) {
+			Liferay.Util.SessionStorage.setItem(
+				persistedTabKey,
+				id,
+				Liferay.Util.SessionStorage.TYPES.PERSONALIZATION
+			);
+		},
+	};
+})();
+
+function activeTab(item) {
+	const getAriaLabel = () => {
+		const label = dropdownButton.getAttribute('aria-label');
+		const [currentSelectionText] = label.split(':');
+
+		return `${currentSelectionText}: ${item.textContent.trim()}`;
+	};
+
+	tabItems.forEach((tabItem) => {
+		tabItem.setAttribute('aria-selected', false);
+		tabItem.classList.remove('active');
+	});
+
+	dropdownButton.setAttribute('aria-label', getAriaLabel());
+	item.setAttribute('aria-selected', true);
+	item.classList.add('active');
+}
+
+function activeTabPanel(item) {
+	tabPanelItems.forEach((tabPanelItem) => {
+		if (!tabPanelItem.classList.contains('d-none')) {
+			tabPanelItem.classList.add('d-none');
+		}
+	});
+
+	item.classList.remove('d-none');
+}
+
+function handleDropdown(event, item) {
+	event.preventDefault();
+	dropdown.classList.toggle('show');
+
+	const ariaExpanded = dropdownButton.getAttribute('aria-expanded');
+
+	dropdownButton.setAttribute('aria-expanded', ariaExpanded === 'false');
+
+	if (item) {
+		handleDropdownButtonName(item);
+	}
+}
+
+function handleDropdownButtonName(item) {
+	const tabText =
+		item.querySelector('lfr-editable') ||
+		item.querySelector('.navbar-text-truncate');
+
+	if (tabText) {
+		dropdownButton.querySelector('.navbar-text-truncate').innerHTML =
+			tabText.textContent;
+	}
+}
+
+function main() {
+	const tabItemId = tabItems[persistedTab.getId()] ? persistedTab.getId() : 0;
+
+	if (
+		dropdownButton &&
+		tabItemId &&
+		tabItems.length &&
+		tabPanelItems.length
+	) {
+		tabItems.forEach((item, index) => {
+			item.addEventListener('click', (event) => {
+				openTabPanel(event, index);
+
+				dropdownButton.focus();
+			});
+		});
+
+		dropdownButton.addEventListener('click', (event) => {
+			handleDropdown(event);
+		});
+
+		activeTab(tabItems[tabItemId]);
+		activeTabPanel(tabPanelItems[tabItemId]);
+		handleDropdownButtonName(tabItems[tabItemId]);
+	}
+}
+
+function openTabPanel(event, i) {
+	const currentTarget = event.currentTarget;
+	const target = event.target;
+
+	const isEditable =
+		target.hasAttribute('data-lfr-editable-id') ||
+		target.hasAttribute('contenteditable');
+
+	const dropdownIsOpen = JSON.parse(
```

Comment:
Replace with `Boolean(dropdownButton.getAttribute('aria-expanded')`, no need to use JSON.parse